app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -57,6 +56,49 @@
         {% endfor %}
       {% endfor %}
     {% else %}
+      {% import "toolkit/summary-table.html" as summary %}
+        {% call summary.mapping_table(
+        row_bold_borders=True,
+        text_large = True,
+        no_bottom_border=True,
+        caption="If you publish today, you must be aware of the following dates:",
+        caption_visible=True,
+        field_headings=[
+          "Date",
+          "Notice",
+        ],
+        field_headings_visible=False
+      ) %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.field_heading_date("Today", scope='row') }}
+          {{ summary.text("Suppliers can apply and ask questions about your requirements.") }}
+        {% endcall %}
+        {% call summary.row(bold_border=True) %}
+          {{ summary.field_heading_date("Before {}".format(dates.questions_close | shortdateformat), rowspan='3', scope='row') }}
+          {{ summary.text_bold("Details of your question and answer session") }}
+        {% endcall %}
+        {% call summary.row(no_border=True) %}
+          {{ summary.text("Question and answer session information.") }}
```

Comment:
The previous comment got hidden, but this still needs to be replaced with the actual brief data